.github/workflows/prerelease.yml

```diff
@@ -227,7 +227,7 @@ jobs:
       - name : Create MSI installer
         shell: pwsh
         run: |
-          .\scripts\win_msi_build.ps1 -arch ${{ matrix.goarch }} -exporterName ${{ steps.vars.outputs.NAME }} -version ${{ steps.vars.outputs.VERSION }} -exporterGUID ${{ steps.vars.outputs.EXPORTER_GUID }} -upgradeGUID ${{ steps.vars.outputs.UPGRADE_GUID }} -licenseGUID ${{ steps.vars.outputs.LICENSE_GUID }} -configGUID ${{ steps.vars.outputs.CONFIG_GUID }} -pfx_passphrase "$env:PFX_PASSPHRASE"
+          .\scripts\pkg\windows\nri-amd64-installer\win_package.sh -exporterName=${{ steps.vars.outputs.NAME }} -version=${{ steps.vars.outputs.VERSION }} -upgradeGUID=${{ steps.vars.outputs.UPGRADE_GUID }} -pfx_passphrase="$env:PFX_PASSPHRASE"
```

Comment:
I am a bit confused, are we running a `.sh` from a `pwsh` shell? is it intended?

Reply:
before it was a .ps1 file

Reply:
I forgot to update `shell: bash`. I relied on this slack https://newrelic.slack.com/archives/GP2EW7ZMG/p1668096824641939, there you have all the links to the windows-runner specifics supporting it.  However, I found an issue I'm trying to solve thanks to this comment 😅

Reply:
Updated in last commit f1d03c7555e29e57c3af119b79114ca7b9753c77.